fhirpy/base/lib.py

```diff
@@ -119,6 +119,9 @@ def execute(self, path, method='post', **kwargs):
         return self._do_request(method, path, **kwargs)
 
     def _do_request(self, method, path, data=None, params=None):
+        if method == 'patch':
+            self.extra_headers['Content-Type'] = 'application/json-patch+json'
```

Comment:
not sure that it's good idea to override client's headers on this step.

Reply:
The request is constructed only just before it is sent and is needed. So where else could it be?

Reply:
Mutating self.extra_headers affect all future requests. It's better to override this header further
```
r = requests.request(method, url, json=data, headers=headers)
```